ingeniamotion/communication.py

```diff
@@ -800,7 +800,8 @@ def connect_servo_ethercat_interface_ip(
     def scan_servos_ethercat_with_info(
         interface_name: str,
     ) -> OrderedDict[int, SlaveInfo]:
-        """Scan a network adapter to get all connected EtherCAT slaves including slave information.
+        """Scan a network adapter to get all connected EtherCAT
+        slaves including slave information - CoE.
```

Comment:
Scanning the servos present in an ethercat network would also include nodes without CoE. I would remove it here